app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CanTheyUseOurServiceController.scala

```diff
@@ -120,36 +130,91 @@ class CanTheyUseOurServiceController @Inject()(
-            numberOfPropertiesForm
-              .bindFromRequest()
-              .fold(
-                formWithErrors => BadRequest(howManyPropertiesPage(formWithErrors)), { numberOfProperties =>
-                  val newSubscribed = subscribed.copy(
-                    individualTriageAnswers = Some(
-                      individualTriageAnswers.copy(numberOfProperties = Some(numberOfProperties)))
-                    )
-
-                  updateSession(sessionStore, request)(_.copy(journeyStatus = Some(newSubscribed)))
-                    .map {
-                      case Left(e) =>
-                        logger.warn("Could not update session", e)
-                        errorHandler.errorResult()
-
-                      case Right(_) =>
-                        Ok(s"Got number of properties $numberOfProperties")
+  def howManyPropertiesSubmit(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withIndividualTriageAnswers(request) {
+      case (_, subscribed, individualTriageAnswers) =>
+        if (individualTriageAnswers.individualUserType.isEmpty) {
+          Redirect(routes.CanTheyUseOurServiceController.whoIsIndividualRepresenting())
+        } else {
+          numberOfPropertiesForm
+            .bindFromRequest()
+            .fold(
+              formWithErrors => BadRequest(howManyPropertiesPage(formWithErrors)), { numberOfProperties =>
+                val newSubscribed = subscribed.copy(
+                  individualTriageAnswers =
+                    Some(individualTriageAnswers.copy(numberOfProperties = Some(numberOfProperties)))
+                )
+
+                updateSession(sessionStore, request)(_.copy(journeyStatus = Some(newSubscribed)))
+                  .map {
+                    case Left(e) =>
+                      logger.warn("Could not update session", e)
+                      errorHandler.errorResult()
+
+                    case Right(_) =>
+                      numberOfProperties match {
+                        case NumberOfProperties.One =>
+                        Redirect(routes.CanTheyUseOurServiceController.whenWasDisposalDate())
+
+                        case NumberOfProperties.MoreThanOne =>
+                          Ok("multiple disposals not handled yet")
+                      }
+                  }
+
+              }
+            )
+        }
+    }
+  }
+
+  def whenWasDisposalDate(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withIndividualTriageAnswers(request) {
+      case (_, subscribed, individualTriageAnswers) =>
+        if (individualTriageAnswers.numberOfProperties.isEmpty) {
+          Redirect(routes.CanTheyUseOurServiceController.howManyProperties())
+        } else {
+          val today = LocalDate.now(clock)
+          val form = subscribed.individualTriageAnswers
+            .flatMap(_.disposalDate)
+            .fold(disposalDateForm(today))(disposalDateForm(today).fill)
```

Comment:
`today` is UTC
How do you know what timezone the user is in? Say they are overseas residents.

Reply:
I had a chat with Jo and Ash about this from a BA and UR angle. The conclusion was came to was that it's probably best to measure today against the "Europe/London" timezone. Putting it to the JVM's timezone wouldn't work because it would likely set it to the timezone the data centre hosting our service is in.

To use our service you need to have completed the disposal - so the completion date has to be "today" or before. The disposal date must be before the completion date so that must also be today or before. We went through this scenario:
 - completion date is set for 2nd Jan
 - user is in Australia somewhere where the time difference is 12 hours. It is 11pm 1st Jan in the UK and it is 11am 2nd Jan for the user in Australia
- user tries to use our service to start a return
- user enters in their "today" for the completion date (2nd Jan)
- we see that the date is before our today (1st Jan) and display a message which currently says something like "completion date must be today or before"

This would probably cause confusion for the user, so we're going to double check the error message here. But in this scenario, the completion could not have occurred at the time the user was doing it since the disposal must be done in the UK, so no-one would have actually done any of the paperwork necessary for the completion to take place in the scenario above. The above scenario is very unlikely to happen unless someone is super eager to do their return.

Reply:
I changed the clock to be `Clock.system(ZoneId.of("Europe/London"))` now

Reply:
No, the JVM TZ will be set according to the configuration of the VM the JVM lives in. I checked with the infrastructure team and they set it to UTC. This will only ever come an issue if you start doing date calculations inside the CGT service which I don't see anywhere at the moment. But the correct thing to do for now is to be consistent in the timezones. 

Given that the JVM TZ is UTC, all dates entered by the user will be parsed as UTC dates as you are using LocalDate, even if the user may consider the date he is entering to be of some other TZ. So the the `today` value should also be

```scala
val today = LocalDate.now(ZoneOffset.UTC)
```

or even just

```scala
val today = LocalDate.now
```
if you trust the platform.

Reply:
I wouldn't trust the platform - I'd still stick with explicitly stating UTC in our code to make sure. I'll revert to what I had when I created this PR then and use `Clock.systemUTC()` to make sure the LocalDate is always relative to UTC

Reply:
I've changed it back to UTC now